crescent/commands/options.py

```diff
@@ -236,6 +238,19 @@ def option(
     ...
 
 
+@overload
+def option(
+    option_type: Type[int],
+    description: str = ...,
+    *,
+    autocomplete: Optional[AutocompleteCallbackT] = ...,
+    min_value: Optional[int] = ...,
+    max_value: Optional[int] = ...,
+    name: Optional[str] = ...,
+) -> ClassCommandOption[int]:
+    ...
+
+
```

Comment:
We don't need to make new overloads to include autocomplete, since it doesn't affect the return type. Just add them to the existing decorators.

Reply:
change is made